bin/upgrading/update-db.pl

```diff
@@ -701,47 +701,6 @@ sub populate_moods {
     }
 }
 
-sub populate_mogile_conf {
-    # create/update the MogileFS database if we use it
-    return unless defined $LJ::MOGILEFS_CONFIG{hosts};
```

Comment:
Shouldn't we leave this as is, as long as we support the option of using MogileFS?  Maybe just make the inclusion of MogileFS::Admin also dependent on whether MOGILEFS_CONFIG is defined?